src/setuptools_pyproject_migration/__init__.py

```diff
@@ -282,9 +288,7 @@ def _generate(self) -> Pyproject:
 
         urls: Dict[str, str] = dist.metadata.project_urls
         if urls:
-            urls_table = tomlkit.inline_table()
-            urls_table.update(urls)
-            pyproject["project"]["urls"] = urls_table
+            pyproject["project"]["urls"] = urls
```

Comment:
Did we intend to revert the `urls` field here from being an inline table back to a regular table?

Reply:
Yes, since inline tables are one line they get cramped with URLs:

```toml
urls = {Homepage = "https://example.com/repo/example", Docs = "https://docs.example.com", Funding = "https://example.com/funding/", etc = "and so on and so forth"}
```